lib/src/main/java/com/ibm/bluemix/appid/android/internal/config/Config.java

```diff
@@ -40,4 +41,12 @@ public static String getUserProfilesServerUrl (AppID appId) {
 		serverUrl += ATTRIBUTES_ENDPOINT;
 		return serverUrl;
 	}
+
+	public static String getPublicKeysEndpoint (AppID appId) {
+		return Config.getOAuthServerUrl(appId)+PUBLIC_KEYS_ENDPOINT;
```

Comment:
missing spaces between "+"

Reply:
It is used for getting the public keys in the lookUpPublicKey method in TokenManager.